courses/free-courses-en.md

```diff
@@ -2,6 +2,7 @@
 
 * [Algorithms & Data Structures](#algorithms--data-structures)
 * [Android](#android)
+* [Artificial Intelligence](#ai)
```

Comment:
TOC needs be synchronized with their respective section. So use right GH markdown syntax

```suggestion
* [Artificial Intelligence](#artificial-intelligence)
```